Samples/BatchAI/ManageBatchAI.cs

```diff
@@ -15,6 +15,7 @@
 using Microsoft.Azure.Management.Storage.Fluent.Models;
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.File;
+using ExecutionState = Microsoft.Azure.Management.BatchAI.Fluent.Models.ExecutionState;
```

Comment:
Why this change?

Reply:
Compute have same named `ExecutionState` after generate new version.